third_party/transducer/CMakeLists.txt

```diff
@@ -0,0 +1,38 @@
+CMAKE_MINIMUM_REQUIRED(VERSION 3.5)
+
+PROJECT(rnnt_release)
+
+SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -O2")
+
+IF(APPLE)
+    ADD_DEFINITIONS(-DAPPLE)
+ENDIF()
+
+INCLUDE_DIRECTORIES(submodule/include)
```

Comment:
Please do not use `include_directories`, this add the target directory to all the project, which is not necessary.

Use `target_include_directories` instead. Setting it as `PUBLIC` target, you might be able to get rid of the second call to `install`.